meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -192,10 +192,6 @@ INHERIT += "buildhistory"
 # Copy the image license manifest into DEPLOY_DIR_IMAGE
 INHERIT += "deploy-license-manifest"
 
-# Warn if there are available update layers which are not included in the
-# current configuration
-INHERIT += "flex-updates-check"
```

Comment:
Aren't we checking the inclusion of meta-flex-cve using this class?

Reply:
Erk, yes, thanks, well spotted :)

Reply:
Fixed